bodhi/server/models.py

```diff
@@ -1143,25 +1143,34 @@ def comments_since_karma_reset(self):
                 break
             yield comment
 
+    @staticmethod
+    def contains_critpath_component(builds, release_name):
+        """
+        Determines if there is a critpath component in the builds passed in.
+        :param builds: a list of Build objects
+        :return: a boolean determining if the update contains a critpath
+        component
+        """
+        for build in builds:
+            # This function call is cached, so there is no need to optimize
+            # it here.
+            critpath_components = get_critpath_components(
+                release_name.lower(), build.package.type)
```

Comment:
Is ``.type`` really a string here?  I would guess that it is a ``ContentType`` ``EnumSymbol``.  Do you need ``build.package.type.value`` instead?